src/components/modules/VerticalCard/VerticalCard.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import './VerticalCard.css';
+
+const VerticalCard = props => {
+    return (
+        <div className='row'>
+            <div className='col-sm-4'>
+                <div className="media">
+                    <img src={props.imgsrc} class="img-fluid" alt="image" width="200" height="200"/>
```

Comment:
we could give the width and height of the image in css?

Reply:
good point, 
actually it should be a `prop` will fix it in next PR
adding a TODO now